projects/com.oracle.truffle.llvm.test/src/com/oracle/truffle/llvm/test/alpha/BaseSuite.java

```diff
@@ -80,6 +80,7 @@ public void test() throws Exception {
             CaptureOutput.startCapturing();
             int sulongResult = -1;
             try {
+                assert candidate.toAbsolutePath().toFile().exists() : "File " + candidate.toAbsolutePath().toFile() + " does not exist.";
```

Comment:
Maybe we could make an explicit check out of that, and throw an exception if the file does not exists??